test/Microsoft.ML.Sweeper.Tests/Microsoft.ML.Sweeper.Tests.csproj

```diff
@@ -1,10 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <DefineConstants>CORECLR</DefineConstants>
+    <DefineConstants>CORECLR;PORTED</DefineConstants>
   </PropertyGroup>
   <ItemGroup>
     <ProjectReference Include="..\..\src\Microsoft.ML.Sweeper\Microsoft.ML.Sweeper.csproj" />
     <ProjectReference Include="..\Microsoft.ML.TestFramework\Microsoft.ML.TestFramework.csproj" />
+
+    <NativeAssemblyReference Include="FastTreeNative" />
```

Comment:
I didn't find any "fast tree" references in the new test code, what uses this?

Reply:
```TestDeterministicSweeperAsyncParallel``` and ```testSmacSweeper``` uses this